app/actions/ModActionPanel/QuickAction.tsx

```diff
@@ -398,7 +398,10 @@ function Form(
         ToolsOzoneModerationDefs.isModEventLabel(coreEvent)
 
       setModEventType(
-        eventMayNeedEmail && !shouldMoveToNextSubject && canSendEmail
+        eventMayNeedEmail &&
+          !shouldMoveToNextSubject &&
+          canSendEmail &&
+          isSubjectDid
```

Comment:
Is adding `isSubjectDid` as a condition a misc bugfix?

Reply:
yeah this is a fix for the case where moving to email action on a record subject doesn't really work since the subject needs to be a did/account.